dialect/sql/schema/inspect.go

```diff
@@ -65,6 +68,15 @@ func (i *Inspector) Tables(ctx context.Context) ([]*Table, error) {
 	return tables, nil
 }
 
+// skipTable skip system tables
+func (i *Inspector) skipTable(ctx context.Context, name string) bool {
```

Comment:
alternative fix would be to allow `.scanColumn` to handle the "" type but this is such an edge case its probably not worth it